### PR TITLE
add form

### PR DESCRIPTION
**Pull request by GITHIYON49**
create new registartion form

### CODE REVIEW

File: app/(auth)/signin/page.tsx [hunk 1]
```diff
@@ -36,8 +36,7 @@ export default function SignIn() {
               </Link>
 
               <p className="2xl:px-20">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit
-                suspendisse.
+                jenish
```
Comment (karthik1704) [inline app/(auth)/signin/page.tsx:39]: replace jenish to this " Lorem ipsum dolor sit amet, consectetur adipiscing elit suspendisse."

File: app/dashboard/registrations/new/page.tsx [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+                    />
+                  </div>
+
+                  <div className="w-full xl:w-1/2">
+                    <label className="mb-2.5 block text-black dark:text-white">
+                      Test Type
+                    </label>
+                    <input
+                      type="text"
+                      name="test-type"
```
Comment (karthik1704) [inline app/dashboard/registrations/new/page.tsx:46]: test_type not test-type and change all name contains - to _

File: app/dashboard/registrations/new/page.tsx [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+                    />
+                  </div>
+
+                  <div className="w-full xl:w-1/2">
+                    <label className="mb-2.5 block text-black dark:text-white">
+                      Batch No / Lot No
+                    </label>
+                    <input
+                      type="text"
+                      name="batch-lot--no"
```
Comment (karthik1704) [inline app/dashboard/registrations/new/page.tsx:248]: double dash?

File: app/dashboard/registrations/new/page.tsx [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+                
+                <div className="mb-4.5 flex flex-col gap-6 xl:flex-row">
+                  <div className="w-full xl:w-1/2">
+                    <label className="mb-2.5 block text-black dark:text-white">
+                      Date Of Registration
+                    </label>
+                    <input
+                      type="date"
+                      name="date-of-regi"
+                      placeholder="Enter Date Of Registration"
```
Comment (karthik1704) [inline app/dashboard/registrations/new/page.tsx:345]: full name date_of_registration

File: app/dashboard/registrations/new/page.tsx [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+
+                
+                <div className="mb-4.5 flex flex-col gap-6 xl:flex-row">
+                  <div className="w-full xl:w-1/2">
+                    <label className="mb-2.5 block text-black dark:text-white">
+                      Date Of Recived
+                    </label>
+                    <input
+                      type="date"
+                      name="date-of-rec"
```
Comment (karthik1704) [inline app/dashboard/registrations/new/page.tsx:371]: full name date_of_recived

**Review by karthik1704 — CHANGES_REQUESTED** [review]

File: app/dashboard/registrations/new/page.tsx (hunk 5 above)
Comment (GITHIYON49) [reply app/dashboard/registrations/new/page.tsx:371]: kk bro

**Review by karthik1704 — APPROVED** [review]

**Comment by karthik1704** [comment]
Good Work!.

**Pull request merged**